modules/local/exomedepth/merge_cnv/main.nf

```diff
@@ -2,24 +2,40 @@
 process CNV_MERGE {
     tag "$meta"
 
+    container "quay.io/biocontainers/coreutils:9.3"
```

Comment:
```suggestion
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/coreutils:9.3' :
        'biocontainers/coreutils:9.3' }"
```

This makes sure that the singularity containers is used in singularity instead of having to convert the docker containers to singularity (which can cause issues sometimes)